### PR TITLE
Make the who uses doctrine section configurable and add a functional …

### PR DESCRIPTION
**Pull request by jwage**
Still time to add @asgrim :)

### CODE REVIEW

**Comment by Majkl578** [comment]
The projects I added in #224 were deliberately picked pseudo-randomly (popularity, downloads) from reverse dependencies on Packagist. Also I didn't want to (and would like not to) include any non-OSS projects here on homepage - we may create separate page for those if desired.

I don't know what project(s) @asgrim meant though.

**Comment by jwage** [comment]
I was just joking about @asgrim 😁

File: config/config.yml [hunk 1]
```diff
@@ -16,3 +16,47 @@ parameters:
...
+            name: PHPUnit
+            url: https://phpunit.de/
+        -
+            name: Prophecy
+            url: https://github.com/phpspec/prophecy
+        -
+            name: phpbench
+            url: https://github.com/phpbench/phpbench
+        -
+            name: Typo3
```
Comment (Majkl578) [inline config/config.yml]: Can you please change this to `TYPO3`? I got this one wrong.

Comment (jwage) [reply config/config.yml]: Done.

**Pull request merged**